### PR TITLE
remove temporary directory for unit test with TestUnpackLayerDuplicateEntries

### PR DESCRIPTION
**Pull request by TrumanLing**
Signed-off-by: Ling FaKe <lingfake@huawei.com>

### CODE REVIEW

**Comment by runcom** [comment]
LGTM

File: image/manifest_test.go [hunk 1]
```diff
@@ -24,6 +24,7 @@ func TestUnpackLayerDuplicateEntries(t *testing.T) {
 	}
 	defer f.Close()
 	defer os.RemoveAll(tarfile)
```
Comment (jonboulle) [inline image/manifest_test.go]: Can you just remove this line now since it's redundant with the dir removeall

Comment (TrumanLing) [inline image/manifest_test.go]: @jonboulle I have done.PTAL

**Comment by jonboulle** [comment]
LGTM

**Comment by wking** [comment]
38c45442 looks good to me.

**Comment by vbatts** [comment]
LGTM

**Pull request merged**